code/modules/mob/living/silicon/robot/drone/drone.dm

```diff
@@ -280,20 +280,6 @@
 				return
 	..()
 
-/mob/living/silicon/robot/drone/Bump(var/atom/movable/AM, yes)
```

Comment:
@Bedshaped Skull said go with the quick fix for now.

Stopping drones from pushing things around is less important than allowing them to charge, without which they die. It was harmful code that didnt serve the intended purpose. Of course it can be re-added in another way, this is a hotfix until that is done. To maintain the game in a playable state